lib/type1.g

```diff
@@ -210,18 +210,8 @@ BIND_GLOBAL("_NEW_TYPE_READONLY", `"NEW_TYPE_READONLY");
 ASS_GVAR(_NEW_TYPE_READONLY, true);
 MakeThreadLocal(_NEW_TYPE_READONLY);
 
```

Comment:
Can't we now completely remove _NEW_TYPE_READONLY? The only place the ever changed it from its default value `true` is gone.

In particular, types now are always read-only, so we should catch any "offenders" that modify types later on eventually.